### PR TITLE
Build improvements

### PR DESCRIPTION
**Pull request by yurloc**

### CODE REVIEW

File: employee-rostering-frontend/package.json [hunk 1]
```diff
@@ -37,13 +37,21 @@
   "scripts": {
     "start": "mvn generate-sources; react-scripts start",
     "build": "mvn generate-sources; react-scripts build",
-    "typecheck": "npx tsc --noEmit",
-    "test": "TZ=UTC react-scripts test --transformIgnorePatterns \"node_modules/(?!(lodash-es))/\"",
+    "typecheck": "tsc --noEmit",
```
Comment (Christopher-Chianelli) [inline employee-rostering-frontend/package.json]: I am assuming this is a deliberate error to verify it fails the build?

File: employee-rostering-frontend/pom.xml [hunk 2]
```diff
@@ -29,16 +29,6 @@
...
-  <properties>
-    <!--
-      Intentionally leaving these properties empty so that the Frontend plugin uses default values and downloads
-      Node.js and npm from https://nodejs.org/dist/. Having these properties here makes it possible to change the URLs
-      using system properties. This is necessary for CI builds running on Red Hat infrastructure.
-    -->
-    <nodeDownloadRoot></nodeDownloadRoot>
-    <npmDownloadRoot></npmDownloadRoot>
-  </properties>
-
```
Comment (Christopher-Chianelli) [inline employee-rostering-frontend/pom.xml]: Why is this removed?

Comment (yurloc) [reply employee-rostering-frontend/pom.xml]: See commit description.

File: employee-rostering-frontend/package.json (hunk 1 above)
Comment (yurloc) [reply employee-rostering-frontend/package.json]: I know we've discussed this on the chat but just for the record: This will actually fix the build on Jenkins where `npx` seems to be missing. Without `npx` in the script, when running `npm run typecheck`, `npm` will look for `tsc` in `node_modules/.bin` where it is provided by the `typescript` dependency.
